tests/unit/test_client.py

```diff
@@ -30,7 +30,7 @@ class ClientTestData:
     )
 
     update_response = Client(
-        id='DEV:GOV:9876:SUB1',
+        id='DEV:GOV:9876',
```

Comment:
Why is this needed?

Reply:
i have deleted it, it's not used in the class